src/wmnds/components/content-tiles/small3/small3.njk

```diff
@@ -0,0 +1,14 @@
+{% macro wmndsContentTilesSmall3(params) %}
+
+<div class="wmnds-content-tile">
+    <div>
+    <h2 class="wmnds-content-tile__title">Where the adventure starts</h2>
+    </div>
+    <div class="wmnds-content-tile--copy">
+    <p>Whatever your plans this holiday, get there with our great range of family tickets and discount vouchers.</p>
+    </div>
+    <div class="wmnds-col-1"><a href="#" class="wmnds-link wmnds-float--right">Read more</a></div>
```

Comment:
Indent and remove a grid refs.

Reply:
What would be a good alternative to use instead of grid ref on 'read more'?